src/scripts.js

```diff
@@ -96,8 +105,9 @@ function toggleHidden(element) {
 }
 
 function collapseRecipe(event) {
+  body.classList.remove('no-scroll')
   createRecipeCards(currentView);
-  if (event.target.id === "recipeModalBackground"){
+  if (event.target.id === "recipeModalBackground" || event.target.id === "closeIcon"){
```

Comment:
Super elegant solution to include the close icon as on option for exiting the modal! Great work @jsahim

Reply:
Wow love this solution! Good work @jsahim

Reply:
This is a great fix!